packages/zowe-explorer/src/command/MvsCommandHandler.ts

```diff
@@ -76,11 +78,8 @@ export class MvsCommandHandler extends ZoweCommandProvider {
             }
         }
         if (!session) {
-            const profiles = Profiles.getInstance();
             const allProfiles: imperative.IProfileLoaded[] = profiles.allProfiles;
```

Comment:
I noticed `profiles.allProfiles` is already typed as `imperative.IProfileLoaded[]`, so we can remove the type annotation on `allProfiles` (line 81). But, it's not a part of this PR's changes so there's no immediate need to remove it, just spotted this while reviewing.